src/reynir_correct/config/GreynirCorrect.conf

```diff
@@ -3582,6 +3582,143 @@ sjálfstæðismaður
 Lenínismi
 Marxismi
 Darwinismi
+Heimsstyrjöld	# Fyrri/Seinni/Síðari Heimsstyrjöld
+Stríð 			# Kalda Stríðið
+Heimsveldi
+Bylting
+Uppreisn
+Meistaradeild
+Kreppa
+Endurreisn
+Jörð
+Tungl
+
+# Holidays
+# Unnecessary to include: föstudagurinn langi
+# Unable to include: annar í páskum, sumardagurinn fyrsti, baráttudagur verkalýðsins, annar í hvítasunnu, frídagur verslunarmanna, fyrsti vetrardagur, dag íslenskrar tungu, annar í jólum
+
+Nýársdagur
+Þrettándi
+Bóndadagur
+Konudagur
+Bolludagur
+Sprengidagur
+Öskudagur
+Páskar
+Pálmasunnudagur
+Skírdagur
+Páskadagur
+Mæðradagur
+Uppstigningardagur
+Hvítasunnudagur
+Hvítasunna
+Sjómannadagur
+Þjóðhátíðardagur
+Verslunarmannahelgi
+Gleðiganga
+Menningarnótt
+Hrekkjavaka
+Feðradagur
+Fullveldisdagur
+Jól
+Aðfangadagur
+Aðfangadagskvöld
+Jóladagur
+Áramót
+Gamlársdagur
+jónsmessa
+valentínusardagur
+þorláksmessa
+
+# Job titles
+# Unable to include: félags- og barnamálaráðherra, fjármála- og efnahagsráðherra, utanríkis- og þróunarsamvinnuráðherra, utanríkis- og þróunarsamvinnuráðherra, umhverfis- og auðlindaráðherra, sjávarútvegs- og landbúnaðarráðherra, mennta- og menningarmálaráðherra, samgöngu- og sveitarstjórnarráðherra, ferðamála-, iðnaðar- og nýsköpunarráðherra, sérstakur saksóknari, sr., hr.
```

Comment:
These can probably be fixed by making the .conf file parser a little more intelligent.

Reply:
...and I'm working on it.